demisto_sdk/commands/upload/uploader.py

```diff
@@ -77,6 +77,11 @@ def __init__(self, input: str, insecure: bool = False, verbose: bool = False):
     def upload(self):
         """Upload the pack / directory / file to the remote Cortex XSOAR instance.
         """
+        if self.demisto_version == "0":
+            click.secho("Could not connect to XSOAR server. Try checking your connection configurations.",
```

Comment:
if the problem is caused by the connection to the server, wouldn't it raise an exception in an earlier stage (when calling the `configure` function of demisto-py)?

Reply:
That's what I thought too, but if you have a look at the function, it doesn't actually do any connecting to the server, only configures a class if you use an API